platform/src/main/java/org/stellar/anchor/platform/utils/AssetValidationUtils.java

```diff
@@ -70,7 +58,19 @@ public static void validateFeeDetails(
     }
   }
 
-  public static void validateAsset(
+  /**
+   * validateAsset will validate if the provided amount has valid values and if its asset is
+   * supported.
+   *
+   * @param amount is the object containing the asset full name and the amount.
+   */
+  public static void validateAssetAmount(
+      String fieldName, AmountAssetRequest amount, AssetService assetService)
+      throws BadRequestException {
+    validateAssetAmount(fieldName, amount, false, assetService);
+  }
+
+  public static void validateAssetAmount(
```

Comment:
minor: Let's add Javadoc for these methods.

Reply:
Improved.